src/main/java/net/openhft/chronicle/core/util/AssertUtil.java

```diff
@@ -126,7 +126,7 @@
      * Performance critical code should use one of the first two schemes devised above to assert invariants.
      * The third, more convenient form, can be used for non-performance critical code.
      */
-    public static final boolean SKIP_ASSERTIONS = true;
+    public static final boolean SKIP_ASSERTIONS = false;
```

Comment:
Can we put it to system property? I can make sure all tests run with this property enabled.

Reply:
Unfortunately, that is not possible because then the compiler cannot collapse the expressions. But we might add a special JAR file before the normal JAR files that has only this class with `SKIP_ASSERTIONS = false;`. Classes found earlier on the CP JARs will take precedence.

Reply:
Maybe a Maven profile?